lib/plugin/instance.rb

```diff
@@ -604,6 +604,15 @@ def each_globbed_asset
     end
   end
 
+  def register_reviewable_type(reviewable_type)
```

Comment:
I noticed in the Akismet plugin you called this with `MyType.name` - what do you think about passing the class in instead of the string?

Reply:
Sounds good!